abna/__init__.py

```diff
@@ -1,16 +1,21 @@
 from cryptography.hazmat.backends import default_backend
 from cryptography.hazmat.primitives.asymmetric.padding import PKCS1v15
 from cryptography.hazmat.primitives.asymmetric.rsa import RSAPublicNumbers
+from fake_useragent import UserAgent
 import copy, requests, sys
 
 BASE = 'https://www.abnamro.nl'
 START = BASE + '/portalserver/mijn-abnamro/mijn-overzicht/overzicht/index.html'
 SERVICE_VERSION = {'x-aab-serviceversion': 'v3'}
 
+
 class Session(object):
     def __init__(self, iban):
         self.iban = iban
         self.session = requests.Session()
+        self.ua = UserAgent().random
+        self.headers = SERVICE_VERSION
+        self.headers["User-Agent"] = self.ua
```

Comment:
I think the best approach here would involve just populating the default headers for the session with the `x-aab-serviceversion` and `User-Agent` header, in that case we can actually simplify the other code in the class and it will do the right thing automatically.